### PR TITLE
feat: rename devnets

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Review by pk910 — APPROVED** [review]

File: roles/generate_kubernetes_config/templates/dugtrio.yaml.j2 [hunk 1]
```diff
@@ -5,6 +5,7 @@ dugtrio:
   image:
     repository: {{ default_tooling_images.dugtrio.split(':') | first }}
     tag: {{ default_tooling_images.dugtrio.split(':') | last }}
+    pullPolicy: Always
```
Comment (pk910) [inline roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: why did you add this again?

**Review by pk910 — CHANGES_REQUESTED** [review]

File: roles/generate_kubernetes_config/templates/dugtrio.yaml.j2 (hunk 1 above)
Comment (barnabasbusa) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: Here we can have this pull policy, just not in the upstream ethereum helm chart.

Comment (barnabasbusa) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: This would roll the updates on devnets

Comment (pk910) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: tbh, I'd leave it with IfNotPresent here.
It's actually the same as with the upstream chart - the way to roll image is providing a new tag, so we do not accidentally introduce new bugs just by restarting the pod.
Default tag name is now "1.0.0", which doesn't change anyway

Comment (pk910) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: can we make the pullPolicy conditional?
So use `Always` if `default_tooling_images.dugtrio.split(':') | last ` is `latest`,  if not use `IfNotPresent`

Comment (pk910) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2:8]: I don't wanna be too picky on that, so I'm gonna approve it now ;)
However, it doesn't really make sense to set it to `IfNotPresent` in the upstream helm chart, just to override it with `Always` in literally all devnets :D
It's then basically the same as if we just set `Always` in the upstream repo..

**Review by pk910 — APPROVED** [review]

File: roles/generate_kubernetes_config/templates/dugtrio.yaml.j2 (hunk 1 above)
Comment (pk910) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2]: Just noticed the change, 
Cool 👍

Comment (barnabasbusa) [reply roles/generate_kubernetes_config/templates/dugtrio.yaml.j2]: assuming that we are the only ones using the helm chart, I agree, but the general idea is that there might be others that use the helm chart, and we wanna make sure its gonna work for them too.
